test/registries.conf

```diff
@@ -1,10 +1,20 @@
 # Note that changing the order here may break tests.
-[registries.search]
-registries = ['docker.io', 'quay.io', 'registry.fedoraproject.org']
+unqualified-search-registries = ['docker.io', 'quay.io', 'registry.fedoraproject.org']
 
 [registries.insecure]
 registries = []
 
 #blocked (docker only)
 [registries.block]
 registries = []
+
+# In Nov. 2020, Docker rate-limits image pulling.  To avoid hitting these
+# limits while tesing, use the provided google cache.  Integration tests
+# make use of the copy of this file under `./test/`, however system tests
+# use the globally installed file.
+# Ref: https://cloud.google.com/container-registry/docs/pulling-cached-images
+[[registry]]
+location="docker.io"
+
+[[registry.mirror]]
+location="mirror.gcr.io"
```

Comment:
```suggestion
```

Reply:
oh, did you change your mind or did I misinterpret https://github.com/containers/podman/pull/7965#discussion_r507538219 ?

Reply:
oh nvm, I see your comment below.